apps/etterlatte-vilkaarsvurdering/src/main/resources/db/migration/V7__legg_til_tabeller_for_vilkaar.sql

```diff
@@ -0,0 +1,46 @@
+CREATE TABLE vilkaarsvurdering_v2
```

Comment:
Tror vi må slenge på en `CREATE EXTENSION IF NOT EXISTS "uuid-ossp";` ellers vil migreringen i dev brekke. Gjelder `uuid_generate_v4`.

Reply:
Trodde det holdt at den var kjørt i et tidligere script - men legger den inn uansett 👍